src/languages/php.js

```diff
@@ -368,6 +353,20 @@ export default function(hljs) {
           2: "variable.language",
         },
       },
+      {
```

Comment:
If you add anymore you may have to start naming them and breaking them out into their own constants. :)

Reply:
This time that's all :). Back then I didn't notice they break each over.